src/js/components/award/financialAssistance/CFDAOverview.jsx

```diff
@@ -5,28 +5,50 @@ import { TooltipWrapper } from 'data-transparency-ui';
 import { CFDAOverviewInfo } from '../shared/InfoTooltipContent';
 
 const propTypes = {
-    cfdaPropgram: PropTypes.string
+    cfdaPropgram: PropTypes.string,
+    cfdaCount: PropTypes.number,
+    jumpToSection: PropTypes.func
 };
 
-const CFDAOverview = ({
-    cfdaPropgram
-}) => (
-    <div className="award-overview__right-section__cfda award-overview-column first award-overview-column__spacing">
-        <h6 className="award-overview-title">
-            CFDA Program / Assistance Listing
-            <TooltipWrapper
-                className="award-section-tt"
-                icon="info"
-                left
-                tooltipComponent={CFDAOverviewInfo} />
-        </h6>
-        <div className="award-overview__body award-overview__cfda">
-            <span>
-                {cfdaPropgram}
-            </span>
-        </div>
-    </div>
-);
+export default class CFDAOverview extends React.Component {
+    constructor(props) {
+        super(props);
+        this.jumpToCFDASection = this.jumpToCFDASection.bind(this);
+    }
+
+    jumpToCFDASection() {
+        this.props.jumpToSection('cfda');
+    }
+    render() {
+        const { cfdaCount, cfdaPropgram } = this.props;
+        return (
+            <div className="award-overview__right-section__cfda award-overview-column first award-overview-column__spacing">
+                <h6 className="award-overview-title">
+                    {cfdaCount === 1 ?
+                        "CFDA Program / Assistance Listing" : "Primary CFDA / Assistance Listing"}
+                    <TooltipWrapper
+                        className="award-section-tt"
+                        icon="info"
+                        left
+                        tooltipComponent={CFDAOverviewInfo} />
+                </h6>
+                <div className="award-overview__body award-overview__cfda">
+                    <span>
+                        {cfdaPropgram}
+                    </span>
+                    <div>
+                        <button
+                            key="cfda"
+                            className="cfda-viz__button"
```

Comment:
you should be able to obtain the style you want with this class name `award-viz__link-text`.